plugins/pa11y/readme.md

```diff
@@ -0,0 +1,81 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process.
+ Pa11y requires Node.js 12+ to run
+
+# Pa11y documentation
+ `https://github.com/pa11y/pa11y#javascript-interface`
```

Comment:
**nitpick**: consumers of the Tool Kit plugin won't be particularly concerned about the Javascript interface, rather a general view of what Pa11y does and how to configure it

```suggestion
 `https://github.com/pa11y/pa11y`
```